otherlibs/dune-rpc/dune_rpc.mli

```diff
@@ -86,6 +86,112 @@ module V1 : sig
     val to_string_absolute : t -> string
   end
 
+  module Ansi_color : sig
+    module RGB8 : sig
+      (** 8-bit RGB color *)
+      type t
+
+      (** [of_int i] creates an 8-bit color from the least significant 8 bits of [i]. *)
+      val of_int : int -> t
+
+      (** [to_int t] returns the 8-bit color as an integer in the range [0, 255]. *)
+      val to_int : t -> int
+    end
+
+    module RGB24 : sig
+      (** 24-bit RGB color (true color) *)
+      type t
+
+      (** [make ~red ~green ~blue] creates a 24-bit color from the given RGB
+          components. Each color component should be in the range [0, 255]. *)
+      val make : red:int -> green:int -> blue:int -> t
+
+      (** [red t] returns the red component of the 24-bit color [t]. *)
+      val red : t -> int
+
+      (** [green t] returns the green component of the 24-bit color [t]. *)
+      val green : t -> int
+
+      (** [blue t] returns the blue component of the 24-bit color [t]. *)
+      val blue : t -> int
+
+      (** [of_int i] creates a 24-bit color from the least significant 24 bits of [i].
+          Each color components consists of 8 bits. *)
+      val of_int : int -> t
+
+      (** [to_int t] returns the 24-bit color as an integer in the range [0, 0xFFFFFF].
+          Each color components consists of 8 bits. *)
+      val to_int : t -> int
```

Comment:
Ditto

Reply:
If I'm making a web client for viewing RPC messages, this might be a useful value to have since colors are hex values usually. (As an example of how this might be useful).